validation/standard/standard-immutables.toml

```diff
@@ -0,0 +1,5 @@
+["op-contracts/v1.4.0"] # Fault Proof contract AST https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
+anchor_state_registry = '{"92829":[{"start":387,"length":32},{"start":828,"length":32},{"start":2296,"length":32}]}'
+delayed_weth = '{"97827":[{"start":831,"length":32},{"start":4133,"length":32}]}'
+fault_dispute_game = '{"93651":[{"start":1926,"length":32},{"start":12628,"length":32}],"93654":[{"start":2891,"length":32},{"start":7484,"length":32},{"start":7822,"length":32},{"start":11526,"length":32},{"start":11660,"length":32},{"start":12171,"length":32},{"start":12470,"length":32}],"93657":[{"start":2724,"length":32},{"start":7283,"length":32},{"start":7577,"length":32},{"start":8183,"length":32},{"start":12437,"length":32},{"start":14230,"length":32},{"start":16139,"length":32},{"start":17960,"length":32},{"start":18262,"length":32},{"start":18519,"length":32},{"start":18732,"length":32}],"93661":[{"start":2673,"length":32},{"start":3936,"length":32},{"start":7722,"length":32},{"start":8329,"length":32},{"start":8424,"length":32},{"start":11254,"length":32},{"start":11320,"length":32}],"93665":[{"start":1204,"length":32},{"start":7866,"length":32},{"start":12866,"length":32},{"start":13719,"length":32}],"93669":[{"start":2189,"length":32},{"start":9639,"length":32},{"start":14792,"length":32}],"93673":[{"start":1319,"length":32},{"start":6476,"length":32},{"start":9344,"length":32},{"start":10730,"length":32},{"start":15892,"length":32}],"93677":[{"start":1555,"length":32},{"start":6026,"length":32},{"start":9704,"length":32}],"93680":[{"start":2590,"length":32},{"start":14649,"length":32}],"93684":[{"start":1723,"length":32},{"start":8123,"length":32},{"start":8230,"length":32},{"start":8281,"length":32}]}'
+mips = '{"92829":[{"start":387,"length":32},{"start":828,"length":32},{"start":2296,"length":32}]}'
```

Comment:
Nit, can you tweak your editor settings so that it formats files with a newline at the end? Just prevents noise where this little character appears and disappears...